src/utils/amortization.utils.ts

```diff
@@ -0,0 +1,78 @@
+import { DAYS_IN_REPAYMENT_PERIOD } from "@consts";
+import { Ratio } from "@credix/credix-client";
+import Big from "big.js";
+import { round } from "./format.utils";
+
+export const calculateMonthlyPayment = (
```

Comment:
add the relevant formulas from this page as comments https://www.investopedia.com/terms/a/amortization.asp#:~:text=How%20to%20Calculate%20Amortization%20of%20Loans&text=You'll%20need%20to%20divide,your%20loan%20term%20by%2012 and have the code reflect them.